_data/accessibility-tests/checkbox.yml

```diff
@@ -0,0 +1,54 @@
+title: Checkbox
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain checkbox requirements. 
+    summary_additional: The checkbox has a visible label above it to indicate its purpose. For example, "Select your state."
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons in a checkbox, all labels have short descriptions that are easy to understand and act on. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.6
+# Keyboard navigation tests
+  - summary: Focus indicator on checkbox is clearly visible.
+    summary_additional: When you use a keyboard to focus on a checkbox group, you will see a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.1
+  - summary: Checkbox focus follows logical content order on desktop.
+    summary_additional: When you tab through the checkbox with a keyboard, the focus stays in order of the content as intended. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.3
+  - summary: Checkbox focus can be moved and options can be selected. 
+    summary_additional: When you use a keyboard to navigate to the checkbox, you are able to move focus and select options, using different keys. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces question before any checkbox and then elements of the checkbox.
+    summary_additional: When listening to the checkbox button using a screen reader, you hear the question related to the checkbox read first. Then you will hear the checkbox group and labels within the checkbox and their checked state. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
```

Comment:
@RachelCorsino 

**issue (blocking):** Looks like this was last tested on 04/05/2024. At that time, [only 3.8.0 had been released](https://github.com/uswds/uswds/releases). It looks like this is true for all the criteria. Can you update all the 3.8.1 references to 3.8.0 in this file?
```suggestion
    version_tested: 3.8.0
```

Reply:
Resolved in [5808ba8](https://github.com/uswds/uswds-site/pull/2803/commits/5808ba81a1e73b90ad5129c8a440f7a27c6ad4a9)